contracts/dtoken/tests/userflows/userflow.liquidation.feature

```diff
@@ -0,0 +1,47 @@
+Feature: User liquidation flow
+
+	Rule: Price for WETH, WNEAR are should be the same 5$
+		Background:
+			Given The borrower, liquidator users, WETH, WNEAR, DWETH, DWNEAR contracts
+			And Digital token DWETH contract with supplied 100 tokens by borrower
+			And Digital token DWNEAR contract with supplied 50 tokens by borrower
+			And Digital token DWETH contract with supplied 100 tokens by liquidator
+			And health_threshold should be equal 150% === 15000 in Ratio format
```

Comment:
@pahomas probably we shouldn't mention this as `Given` because we cannot `set` health factor

Reply:
At this place I mentioned literally health_threshold and e have get/set methods
get_health_threshold
set_health_factor_threshold

Reply:
This And is a continuation of `Given`